src/App/VPSScan/Scan/ScotlandYard.hs

```diff
@@ -9,16 +9,24 @@ where
 
 import App.VPSScan.Types
 import Control.Effect.Diagnostics
-import Control.Monad.IO.Class (MonadIO)
+import Control.Monad.IO.Class
```

Comment:
nit: Unless Connor disagrees, I think we should leave these lines as is. I think the goal is to constrict imports as much as possible so that we document which parts of the import we're actually using.

Reply:
Fair enough. I removed them because whatever linter is set up was claiming they were redundant.